portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,192 @@
+          <p>A react application that helps us to organize our todo-list.</p>
+
+          <h5>React.js</h5>
+          <a href="https://github.com/ankeett/todo" class="btn-link">Link</a>
+        </div>
+
+        <div class="card">
+          <h4>On Progress...</h4>
+        </div>
+        
+      </div>
+      
+  </section>
+
+  <section id="about">
+    <h1>About Me</h1>
+
+    <div class="about-div">
+    <div>
+      <img src="./images/self.png" alt="self"">
+    </div>
+    
+    <div>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Et magnis dis parturient montes nascetur ridiculus mus mauris vitae. Vehicula ipsum a arcu cursus vitae congue mauris rhoncus aenean. Consequat ac felis donec et. Lacus laoreet non curabitur gravida arcu ac tortor dignissim. Quis imperdiet massa tincidunt nunc pulvinar sapien et. Malesuada fames ac turpis egestas. Felis eget velit aliquet sagittis id consectetur purus ut faucibus. Nulla posuere sollicitudin aliquam ultrices sagittis orci a scelerisque. Imperdiet dui accumsan sit amet.
+
+        Erat imperdiet sed euismod nisi porta lorem mollis. Eu volutpat odio facilisis mauris sit. Consequat nisl vel pretium lectus. In iaculis nunc sed augue lacus viverra vitae. Sed augue lacus viverra vitae congue eu consequat. Dolor sit amet consectetur adipiscing elit duis tristique. Sed id semper risus in hendrerit gravida rutrum quisque. Duis ut diam quam nulla porttitor. Porttitor rhoncus dolor purus non. Aliquet porttitor lacus luctus accumsan tortor posuere ac ut. Lectus sit amet est placerat in egestas erat. Consectetur adipiscing elit duis tristique sollicitudin. Nisl nisi scelerisque eu ultrices vitae auctor eu augue ut. Ornare massa eget egestas purus viverra accumsan. Lacus vel facilisis volutpat est velit egestas dui id. Quam id leo in vitae turpis massa sed. Non arcu risus quis varius. Duis at consectetur lorem donec massa sapien faucibus et molestie.</p>
+    </div>
+  </div>
+    
+  </section>
+
+  <section id="contact">
+    <h1>Contact Me</h1>
+    <div class="contact">
+      <form action="/form-handler" method="post">
+      <div class="form-items">
+        <label for="name">Name</label>
+        <input type="text" name="name" id="name" placeholder="Name" required>
+      </div>
+    
+      <div class="form-items">
+        <label for="email">Email</label>
+        <input type="text" name="email" id="email" placeholder="Email" required>
+      </div>
+
+      <div class="form-items">
+        <label for="subject">Subject</label>
+        <input type="text" name="subject" id="subject" placeholder="Subject" required>
+      </div>
+
+      <div class="form-items">
+        <label for="message">Message</label>
+        <textarea type="text" name="message" id="message" placeholder="Message" required></textarea>
+      </div>
+
+      
+      <div class="form-items">
+        <label for="check">I'm not a robot</label>
```

Comment:
I love this :) The "I am not a robot" is a special google thing that determines if you're a robot based on how you interacted with the site and the button itself. A regular checkbox isn't going to do the same :) You can delete this.

Reply:
I just added that for fun ;)